src/sentry/api/endpoints/organization_stats_v2.py

```diff
@@ -29,8 +30,16 @@ class OrganizationStatsEndpointV2(OrganizationEventsEndpointBase):
         }
     }
 
-    @rate_limit_endpoint(limit=20, window=1)
-    def get(self, request: Request, organization) -> Response:
+    def get(self, request: Request, organization: Organization) -> Response:
+        # Redirect from legacy URL to versioned scheme
+        url = reverse(
+            "sentry-api-0-organization-stats", kwargs={"organization_slug": organization.slug}
+        )
+        return redirect(f"{url}?version=1", permanent=True)
```

Comment:
FYI: This won't work for POST requests because they drop auth headers and payloads